docs/source/intro.rst

```diff
@@ -11,20 +11,20 @@ Don't worry if you've not seen, or never even heard of Python_, it's incredibly
 To whet your appetite, here's a quick (complete) sample script that shows some of the power of Python Script.::
    
    editor.replace("old", "new")
-   editor.pyreplace(r"^Code: ([A-Z]{4,8})", r"The code is \1")
-
-   notepad.runMenuCommand("TextFX Tools", "Delete Blank Lines")
+   
+   notepad.runMenuCommand("Line Operations", "Remove Empty Lines")

```

Comment:
This assumes that there is a `Line Operations` menu item that has a `Remove Empty Lines` submenu item, which is not the case. Instead `notepad.menuCommand(MENUCOMMAND.EDIT_REMOVEEMPTYLINES)` should be used.

Reply:
@Ekopalypse , I double-checked that that menu item I used IS present in Npp 8.5.6 (same as in 8.5.4) by default.

Reply:
However, it may be absent with other languages than English. So then I can add there note to use your command if user's menu language is not English.

Reply:
The runMenuCommand method expects a main menu item like "File, Edit, Search, View ..." as first parameter and a subitem that is directly a child of this main menu item.
Does the call you mentioned actually work for you? With PS3 it doesn't and to be honest, I'd be surprised if it worked with PS2.

Reply:
> Does the call you mentioned actually work for you?  

Yes, it does! (PS2). The old example there, with ("TextFX Tools", "Delete Blank Lines"), didn't work because THAT was not present in the menu at all. 
I have verified each of the examples in the doc.
How else do you think I would have dared to make doc changes ? :)

Reply:
I tried it again after changing the localization to a different language (ex, Romanian) and the command strings in that language, respectively, . At first it didn't work, then fixed it with putting 
`# coding=utf-8` 
at the top of the script. 
So maybe that was the issue in your case? 
Or, maybe you can try setting the 3rd parameter, `refreshCache`, to `True`, as explained in the doc.

I suspect why it works (with PS2, at least) is that 1st argument is searched not only in the main-menu entries (file, edit etc), but also sub-entries, as listed in the language.xml files. 

In any case...

The doc says: 
> For built-in menus use `notepad.menuCommand()`, for non built-in menus (e.g. TextFX and macros you’ve defined), use `notepad.runMenuCommand(menuName, menuOption)`.  

because of efficiency reasons, I understand. 

So I'll replace with `notepad.menuCommand(MENUCOMMAND.EDIT_REMOVEEMPTYLINES)`, as you suggested.